packages/db/migrations/083_exam_config_distributions_parent_softdelete.sql

```
@@ -0,0 +1,57 @@
+--
+-- Postgres has no CREATE OR REPLACE POLICY; DROP + CREATE mirrors the
+-- precedent in 20260411000005_exam_config_security_fixes.sql.
+
+DROP POLICY IF EXISTS admin_select_exam_distributions ON exam_config_distributions;
+CREATE POLICY admin_select_exam_distributions ON exam_config_distributions
+  FOR SELECT TO authenticated
+  USING (
+    EXISTS (
+      SELECT 1 FROM exam_configs ec
+      WHERE ec.id = exam_config_distributions.exam_config_id
+        AND ec.organization_id = (SELECT organization_id FROM users WHERE id = auth.uid())
```

SQLFluff (4.2.1)

[error] The 'WHERE' keyword should always start a new line.

(LT14)